docs/tempo/website/configuration/ingestion-limit.md

```diff
@@ -50,7 +50,7 @@ To configure new ingestion limits that applies to all tenants of the cluster:
     overrides:
         ingestion_burst_size: 50000
         ingestion_rate_limit: 50000
-        max_spans_per_trace: 50000
+        max_bytes_per_trace: 5_000_000
```

Comment:
Ultra nit: `5000000` instead of `5_000_000`
(this happens in multiple places)

That way this value has the same format that `ingestion_rate_limit` and the rest of big numbers.

Reply:
I actually prefer the underscores and I think showing them in the examples is helpful to other operators.

Reply:
I didn't know about this format until I saw it in a recent PR from Marty, and I honestly like it more, so maybe we can eventually move the other numbers into this format?

Reply:
Yup, I prefer this format too. It's easier to read!

> so maybe we can eventually move the other numbers into this format?
+1